docs/Workshop_Areas/Lab/biolab-usage-guidelines.md

```diff
@@ -0,0 +1,378 @@
+as such, thermally resistant gloves are provided.
+
+One of the primary uses of the microwave is heating of agarose for the
+casting of electrophoresis gels. Users should be aware that prior,
+possibly less careful, microwave operators may have inadvertently
+contaminated the microwave with biological material and/or hazardous
+chemicals, operation of the microwave using gloves and goggles is
+therefore recommended in any case.
+
+More information on microwave usage is available in
+[LBL04008](lbl04008.md).
+
+we currently dont have a microwave and is on the shopping list
+
+
+## PCR Machine
+
+The Polymerase Chain Reaction (PCR) machine, also known as a thermal
+cycler, is used to amplify specific sections of template DNA either for
+diagnostic or molecular cloning purposes. Information regarding the
+current state of the machine including current program and temperature
+is displayed on the LCD on the front panel of this machine.
+
+The heating block and underside lid of this machine will typically reach
+temperatures in the region of 95-100°C in a typical amplification
+program and caution should be exercised when placing and removing
+samples.
+
+More information on usage of the (see below )thermal cycler is available in
+[LBL07004](lbl07004.md) and the [technical
+documentation](https://wiki.Manchester.hackspace.org.uk/w/images/6/6c/Gensoft.pdf)[^4].
+
+A thermo Cycler is being build and will be finished soon 
+
+## HEPA flow cabinet/area
+
+The HEPA-filtered laminar flow unit allows us to work in sterile air in
+order to prevent contamination of our work, such as petri dishes and
+broths, with other micro-organisms.
+
+It should be noted that the laminar flow functions in such a way as to
+protect the user's work rather than the user, and so caution must still
+be exercised by the user when maintain the aseptic conditions of
+materials used within the flow area, and in disposal of consumables to
+assist in containment of biomaterial.
+
+For more information on aseptic technique, see [LBL04005](lbl04005.md).
+
+## Autoclave
+
+The autoclave is used to destroy any potential micro-organisms that
+might contaminate media, reagents and consumables to be used aseptically
+in micro- and molecular biological procedures. High temperature and
+pressure is used to kill contaminants. The high temperature and pressure
+is accompanied by a vacuum cycle.
+
+The main risks to the user are heat of the metal pressurised unit during
+sterilisation and any vented steam. It is recommended that the autoclave
+and contents are allowed to cool for a while before handling.
+
+The correct procedures for using the autoclave/steriliser to prepare
+media and destroy GM waste are described in [LBL04002](lbl04002.md),
+and [LBL04006](lbl04006.md).
+
+## Sink area
+
+The sink area should remain clear. All glassware should be kept clean
+and out of the way.
+
+The sink is connected to the municipal drain. No biological material
```

Comment:
Organic chemicals can't be discharged to drains, for example isopropanol or acetone, as they form flammable vapour mixtures in the drains.
Our limited company status requires us to collect waste solvents that haven't evaporated, and dispose of accordingly.
Waste acetone, or acetone and water mixtures, should be stored separately from other organic solvent waste to prevent formation of explosive peroxide mixtures.
Waste organic and inorganic solvent mixtures should also be stored separately. Waste inorganic solvents may be neutralised with bicarbonate of soda/sodium bicarbonate and discharged into the public drains, this will need to be verified professionally.